packages/artifact/src/internal/upload-http-client.ts

```diff
@@ -406,14 +407,18 @@ export class UploadHttpClient {
     isGzip: boolean,
     totalFileSize: number
   ): Promise<boolean> {
+    // open a new stream and read it to compute the digest
```

Comment:
10 lines up you can add another `@param` with a comment

Reply:
it's not a new param though 🤔

Reply:
Whoops, digest in the utils file